var/spack/repos/builtin/packages/tfel/package.py

```diff
@@ -56,7 +56,7 @@ class Tfel(CMakePackage):
     version("4.0.2", sha256="f5c8a285e00f334fd3e1a95f9a393fed393990ee827dae3766da1decfaa1074e")
     version("4.0.1", sha256="f54741b7e654cb12511ca68c6494a4789ba41b5ada4cd345ad2bc7da631309d1")
     version("4.0.0", sha256="7a0c32c8a9cd2fd65cbcb54fff802f303665d7cba5d46f92ff3d55f057c92845")
-    version("3.4.5", sha256="8092db9571df3d256a56ff269691a871a68b9b19ce163461531879a36e05a100")
+    version("3.4.5", sha256="064d6926106e0052829182087a025f58fc3e98dfb69967e0795d9cdb4e1500b9")
```

Comment:
Do we know why this changed?

Reply:
Yes. I have redone the tar ball for a small typo